openshift/openshift-ansible-contrib
# Support multiple private networks for static inventory

#### What does this PR do?
Support multiple private networks for static inventory / SSH configs generation.
Add openstack_private_network_name to filter by a wanted private network.

#### How should this be manually tested?
Deploy with multiple private networks (update heat template for Nova servers), choose one as an admin/control net and define it as the `openstack_private_network_name`. Finish e2e, it should pass. It should not break things when deploying with a single private network as well.

#### Is there a relevant Issue open for this?
Closes https://github.com/openshift/openshift-ansible-contrib/issues/589

#### Who would you like to review this?
cc: @tomassedovic @oybed  PTAL
